posts/2022-06-01-RLS-deprecation.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "RLS Deprecation"
+author: The Rust Dev Tools Team
+---
+
+The Rust Language Server (RLS) is being deprecated in favor of [rust-analyzer](https://rust-analyzer.github.io/).
```

Comment:
```suggestion
The Rust Language Server (RLS) is being deprecated in favor of [rust-analyzer](https://github.com/rust-lang/rust-analyzer).
```

I'd consider using using the projects GitHub here, to emphasis that it has moved locations.